cardano-api/src/Cardano/Api/SerialiseRaw.hs

```diff
@@ -26,6 +26,9 @@ class (HasTypeProxy a, Typeable a) => SerialiseAsRawBytes a where
   serialiseToRawBytes :: a -> ByteString
 
   deserialiseFromRawBytes :: AsType a -> ByteString -> Maybe a
+  deserialiseFromRawBytes asType bs = rightToMaybe $ eitherDeserialiseFromRawBytes asType bs
+
+  eitherDeserialiseFromRawBytes :: AsType a -> ByteString -> Either String a
```

Comment:
New method `eitherDeserialiseFromRawBytes`.  Old method `deserialiseFromRawBytes` delegates to this.